config/locales/en.yml

```diff
@@ -54,9 +54,9 @@ en:
       inactive_families: "Inactive Families"
       all_families: "All families"
       buttons:
-        active: "Active"
-        inactive: "Inactive"
-        all: "All"
+        active: "ACTIVE"
+        inactive: "INACTIVE"
+        all: "ALL"
```

Comment:
No need to make these all caps here, when we can define that as a stylesheet rule.